fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -121,6 +121,9 @@ public enum DbState {
     @SerializedName(value = "dbProperties")
     private DatabaseProperty dbProperties = new DatabaseProperty();
 
+    @SerializedName(value = "binlogConfig")
+    private BinlogConfig binlogConfig = new BinlogConfig();
```

Comment:
Can it be put into `dbProperties`?

Reply:
I will use it frequently，so maybe cache the binlogConfig is a btter choice.